comment_scan.rb

```diff
@@ -121,8 +121,8 @@ def restart(num_to_post, bundle)
             end
           else
             if reply_args.length > 2 #They're not trying to give a command
-              #Maybe make conversation back (20% chance)
-              cb.say ":#{msg.id} #{random_response()}", room_id if rand() > 0.8
+              #Maybe make conversation back (33% chance)
+              cb.say ":#{msg.id} #{random_response()}", room_id if rand() > 0.67
```

Comment:
I definitely should've caught this before, but as a style thing, you can just write `rand` or `random_response`. You don't need the parens if they're empty.